sdk/reservations/Microsoft.Azure.Management.Reservations/src/Microsoft.Azure.Management.Reservations.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Reservations</PackageId>
 		<Description>Microsoft.Azure.Management.Reservations Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.Reservations</AssemblyName>
-		<Version>1.11.0-preview</Version>
+		<Version>1.21.0-preview</Version>
     <PackageTags>Reservations;</PackageTags>
 		<PackageReleaseNotes>
 			Enhancing Patch API for RI renewal.
```

Comment:
Release notes needs to be updated too

Reply:
Push a new iteration, update version as 1.12 add release notes

Reply:
updated. Is there any other file needs to be changed?